### PR TITLE
Applied fixes from SideCI

### PR DESCRIPTION
**Pull request by vexus2**
Applies code style fixes from an codesniffer by [SideCI](https://www.sideci.com).

### CODE REVIEW

File: app/Console/cake.php [hunk 1]
```diff
@@ -18,29 +18,29 @@
...
 }
 
 $dispatcher = 'Cake' . DS . 'Console' . DS . 'ShellDispatcher.php';
 
 if (function_exists('ini_set')) {
-	$root = dirname(dirname(dirname(__FILE__)));
-	$appDir = basename(dirname(dirname(__FILE__)));
-	$install = $root . DS . 'lib';
-	$composerInstall = $root . DS . $appDir . DS . 'Vendor' . DS . 'cakephp' . DS . 'cakephp' . DS . 'lib';
+    $root = dirname(dirname(dirname(__FILE__)));
```
Comment (ghost) [inline app/Console/cake.php:27]: **[code_sniffer]**
- Equals sign not aligned with surrounding assignments; expected 12 spaces but found 1 space

File: app/Console/cake.php [hunk 2]
```diff
@@ -18,29 +18,29 @@
...
+    $appDir = basename(dirname(dirname(__FILE__)));
```
Comment (ghost) [inline app/Console/cake.php:28]: **[code_sniffer]**
- Equals sign not aligned with surrounding assignments; expected 10 spaces but found 1 space

File: app/Config/Schema/db_acl.php [hunk 3]
```diff
@@ -23,17 +23,19 @@
  * cake schema run create DbAcl
  *
  */
-class DbAclSchema extends CakeSchema {
+class DbAclSchema extends CakeSchema
```
Comment (ghost) [inline app/Config/Schema/db_acl.php:26]: **[phpmd]**
- The property $aros_acos is not named in camelCase.

File: app/Config/Schema/db_acl.php [hunk 4]
```diff
@@ -23,17 +23,19 @@
...
+    /**
  * Before event.
  *
  * @param array $event The event data.
  * @return bool Success
  */
-	public function before($event = array()) {
-		return true;
-	}
+    public function before($event = array())
```
Comment (ghost) [inline app/Config/Schema/db_acl.php:35]: **[phpmd]**
- Avoid unused parameters such as '$event'.

File: app/Config/Schema/db_acl.php [hunk 5]
```diff
@@ -41,62 +43,62 @@ public function before($event = array()) {
  * @param array $event The event data.
  * @return void
  */
-	public function after($event = array()) {
-	}
+    public function after($event = array())
```
Comment (ghost) [inline app/Config/Schema/db_acl.php:46]: **[phpmd]**
- Avoid unused parameters such as '$event'.

File: app/Config/Schema/i18n.php [hunk 6]
```diff
@@ -35,9 +36,10 @@ class I18nSchema extends CakeSchema {
  * @param array $event Schema object properties
  * @return bool Should process continue
  */
-	public function before($event = array()) {
-		return true;
-	}
+    public function before($event = array())
```
Comment (ghost) [inline app/Config/Schema/i18n.php:39]: **[phpmd]**
- Avoid unused parameters such as '$event'.

File: app/Config/Schema/i18n.php [hunk 7]
```diff
@@ -45,17 +47,17 @@ public function before($event = array()) {
...
-	public $i18n = array(
-		'id' => array('type' => 'integer', 'null' => false, 'default' => null, 'length' => 10, 'key' => 'primary'),
-		'locale' => array('type' => 'string', 'null' => false, 'length' => 6, 'key' => 'index'),
-		'model' => array('type' => 'string', 'null' => false, 'key' => 'index'),
-		'foreign_key' => array('type' => 'integer', 'null' => false, 'length' => 10, 'key' => 'index'),
-		'field' => array('type' => 'string', 'null' => false, 'key' => 'index'),
-		'content' => array('type' => 'text', 'null' => true, 'default' => null),
-		'indexes' => array('PRIMARY' => array('column' => 'id', 'unique' => 1), 'locale' => array('column' => 'locale', 'unique' => 0), 'model' => array('column' => 'model', 'unique' => 0), 'row_id' => array('column' => 'foreign_key', 'unique' => 0), 'field' => array('column' => 'field', 'unique' => 0))
-	);
+    public function after($event = array())
```
Comment (ghost) [inline app/Config/Schema/i18n.php:50]: **[phpmd]**
- Avoid unused parameters such as '$event'.

File: app/Config/Schema/sessions.php [hunk 8]
```diff
@@ -24,9 +24,10 @@
  * cake schema run create Sessions
  *
  */
-class SessionsSchema extends CakeSchema {
+class SessionsSchema extends CakeSchema
```
Comment (ghost) [inline app/Config/Schema/sessions.php:27]: **[phpmd]**
- The property $cake_sessions is not named in camelCase.

File: app/Config/Schema/sessions.php [hunk 9]
```diff
@@ -34,9 +35,10 @@ class SessionsSchema extends CakeSchema {
  * @param array $event Schema object properties
  * @return bool Should process continue
  */
-	public function before($event = array()) {
-		return true;
-	}
+    public function before($event = array())
```
Comment (ghost) [inline app/Config/Schema/sessions.php:38]: **[phpmd]**
- Avoid unused parameters such as '$event'.

File: app/Config/Schema/sessions.php [hunk 10]
```diff
@@ -44,14 +46,14 @@ public function before($event = array()) {
...
-	public function after($event = array()) {
-	}
-
-	public $cake_sessions = array(
-		'id' => array('type' => 'string', 'null' => false, 'key' => 'primary'),
-		'data' => array('type' => 'text', 'null' => true, 'default' => null),
-		'expires' => array('type' => 'integer', 'null' => true, 'default' => null),
-		'indexes' => array('PRIMARY' => array('column' => 'id', 'unique' => 1))
-	);
+    public function after($event = array())
```
Comment (ghost) [inline app/Config/Schema/sessions.php:49]: **[phpmd]**
- Avoid unused parameters such as '$event'.

**Comment by sumyapp** [comment]
Click here if you want to create rubocop's auto-correct Pull Request 

https://www.sideci.com/github_repositories/31297639/auto_correct/rubocop/pull_request/130/branch/sideci/20160202062925_codesniffer

File: app/Console/cake.php [hunk 11]
```diff
@@ -18,29 +18,29 @@
...
+    $install = $root . DS . 'lib';
```
Comment (ghost) [inline app/Console/cake.php:29]: **[code_sniffer]**
- Equals sign not aligned with surrounding assignments; expected 9 spaces but found 1 space
- Concat operator must not be surrounded by spaces
- Concat operator must not be surrounded by spaces

File: app/Console/cake.php [hunk 12]
```diff
@@ -18,29 +18,29 @@
...
+    $composerInstall = $root . DS . $appDir . DS . 'Vendor' . DS . 'cakephp' . DS . 'cakephp' . DS . 'lib';
```
Comment (ghost) [inline app/Console/cake.php:30]: **[code_sniffer]**
- Concat operator must not be surrounded by spaces
- Concat operator must not be surrounded by spaces
- Concat operator must not be surrounded by spaces
- Concat operator must not be surrounded by spaces
- Concat operator must not be surrounded by spaces
- Concat operator must not be surrounded by spaces
- Concat operator must not be surrounded by spaces
- Concat operator must not be surrounded by spaces
- Concat operator must not be surrounded by spaces
- Concat operator must not be surrounded by spaces

File: app/Console/cake.php [hunk 13]
```diff
@@ -18,29 +18,29 @@
...
 
-	// the following lines differ from its sibling
-	// /lib/Cake/Console/Templates/skel/Console/cake.php
-	if (file_exists($composerInstall . DS . $dispatcher)) {
-		$install = $composerInstall;
-	}
+    // the following lines differ from its sibling
```
Comment (ghost) [inline app/Console/cake.php:32]: **[code_sniffer]**
- Inline comments must start with a capital letter

File: app/Console/cake.php [hunk 14]
```diff
@@ -18,29 +18,29 @@
...
+    // /lib/Cake/Console/Templates/skel/Console/cake.php
```
Comment (ghost) [inline app/Console/cake.php:33]: **[code_sniffer]**
- Inline comments must end in full-stops, exclamation marks, or question marks

File: app/Console/cake.php [hunk 15]
```diff
@@ -18,29 +18,29 @@
...
+    if (file_exists($composerInstall . DS . $dispatcher)) {
```
Comment (ghost) [inline app/Console/cake.php:34]: **[code_sniffer]**
- Implicit true comparisons prohibited; use === TRUE instead
- Concat operator must not be surrounded by spaces
- Concat operator must not be surrounded by spaces

File: app/Console/cake.php [hunk 16]
```diff
@@ -18,29 +18,29 @@
...
+        $install = $composerInstall;
+    }
 
-	ini_set('include_path', $install . PATH_SEPARATOR . ini_get('include_path'));
-	unset($root, $appDir, $install, $composerInstall);
+    ini_set('include_path', $install . PATH_SEPARATOR . ini_get('include_path'));
```
Comment (ghost) [inline app/Console/cake.php:38]: **[code_sniffer]**
- Concat operator must not be surrounded by spaces
- Concat operator must not be surrounded by spaces

File: app/Config/acl.php [hunk 17]
```diff
@@ -115,8 +115,8 @@
  * to the roles you defined in the roles configuration.
  */
 $config['map'] = array(
-	'User' => 'User/username',
-	'Role' => 'User/group_id',
+    'User' => 'User/username',
```
Comment (ghost) [inline app/Config/acl.php:118]: **[code_sniffer]**
- Array key not aligned correctly; expected 18 spaces but found 4

File: app/Config/acl.php [hunk 18]
```diff
@@ -115,8 +115,8 @@
...
+    'Role' => 'User/group_id',
```
Comment (ghost) [inline app/Config/acl.php:119]: **[code_sniffer]**
- Array key not aligned correctly; expected 18 spaces but found 4

File: app/Config/acl.php [hunk 19]
```diff
@@ -124,22 +124,22 @@
...
 
 /**
  * rule configuration
  */
 $config['rules'] = array(
-	'allow' => array(
-		'*' => 'Role/admin',
-	),
-	'deny' => array(),
+    'allow' => array(
```
Comment (ghost) [inline app/Config/acl.php:141]: **[code_sniffer]**
- Array key not aligned correctly; expected 20 spaces but found 4
- Multi-line array contains a single value; use single-line array instead

File: app/Config/acl.php [hunk 20]
```diff
@@ -124,22 +124,22 @@
...
+        '*' => 'Role/admin',
+    ),
```
Comment (ghost) [inline app/Config/acl.php:143]: **[code_sniffer]**
- Closing parenthesis not aligned correctly; expected 15 space(s) but found 4

File: app/Config/acl.php [hunk 21]
```diff
@@ -124,22 +124,22 @@
...
+    'deny' => array(),
```
Comment (ghost) [inline app/Config/acl.php:144]: **[code_sniffer]**
- Array key not aligned correctly; expected 20 spaces but found 4

File: app/Config/core.php [hunk 22]
```diff
@@ -48,11 +48,11 @@
  *
  * @see ErrorHandler for more information on error handling and configuration.
  */
-	Configure::write('Error', array(
-		'handler' => 'ErrorHandler::handleError',
-		'level' => E_ALL & ~E_DEPRECATED,
-		'trace' => true
-	));
+    Configure::write('Error', array(
```
Comment (ghost) [inline app/Config/core.php:51]: **[code_sniffer]**
- Opening parenthesis of a multi-line function call must be the last content on the line
- Only one argument is allowed per line in a multi-line function call

File: app/Config/core.php [hunk 23]
```diff
@@ -48,11 +48,11 @@
...
+        'handler' => 'ErrorHandler::handleError',
```
Comment (ghost) [inline app/Config/core.php:52]: **[code_sniffer]**
- Array key not aligned correctly; expected 31 spaces but found 8

File: app/Config/core.php [hunk 24]
```diff
@@ -48,11 +48,11 @@
...
+        'level' => E_ALL & ~E_DEPRECATED,
```
Comment (ghost) [inline app/Config/core.php:53]: **[code_sniffer]**
- Array key not aligned correctly; expected 31 spaces but found 8

File: app/Config/core.php [hunk 25]
```diff
@@ -48,11 +48,11 @@
...
+        'trace' => true
```
Comment (ghost) [inline app/Config/core.php:54]: **[code_sniffer]**
- Array key not aligned correctly; expected 31 spaces but found 8

File: app/Config/core.php [hunk 26]
```diff
@@ -48,11 +48,11 @@
...
+    ));
```
Comment (ghost) [inline app/Config/core.php:55]: **[code_sniffer]**
- Closing parenthesis not aligned correctly; expected 30 space(s) but found 4
- Closing parenthesis of a multi-line function call must be on a line by itself

File: app/Console/cake.php (hunk 1 above)
Comment (sumyapp) [inline app/Console/cake.php:27]: **[staging][code_sniffer]**
- Equals sign not aligned with surrounding assignments; expected 12 spaces but found 1 space

File: app/Console/cake.php (hunk 2 above)
Comment (sumyapp) [inline app/Console/cake.php:28]: **[staging][code_sniffer]**
- Equals sign not aligned with surrounding assignments; expected 10 spaces but found 1 space

File: app/Console/cake.php (hunk 11 above)
Comment (sumyapp) [inline app/Console/cake.php:29]: **[staging][code_sniffer]**
- Equals sign not aligned with surrounding assignments; expected 9 spaces but found 1 space
- Concat operator must not be surrounded by spaces
- Concat operator must not be surrounded by spaces

File: app/Console/cake.php (hunk 12 above)
Comment (sumyapp) [inline app/Console/cake.php:30]: **[staging][code_sniffer]**
- Concat operator must not be surrounded by spaces
- Concat operator must not be surrounded by spaces
- Concat operator must not be surrounded by spaces
- Concat operator must not be surrounded by spaces
- Concat operator must not be surrounded by spaces
- Concat operator must not be surrounded by spaces
- Concat operator must not be surrounded by spaces
- Concat operator must not be surrounded by spaces
- Concat operator must not be surrounded by spaces
- Concat operator must not be surrounded by spaces

File: app/Console/cake.php (hunk 13 above)
Comment (sumyapp) [inline app/Console/cake.php:32]: **[staging][code_sniffer]**
- Inline comments must start with a capital letter

File: app/Console/cake.php (hunk 14 above)
Comment (sumyapp) [inline app/Console/cake.php:33]: **[staging][code_sniffer]**
- Inline comments must end in full-stops, exclamation marks, or question marks

File: app/Console/cake.php (hunk 15 above)
Comment (sumyapp) [inline app/Console/cake.php:34]: **[staging][code_sniffer]**
- Implicit true comparisons prohibited; use === TRUE instead
- Concat operator must not be surrounded by spaces
- Concat operator must not be surrounded by spaces

File: app/Console/cake.php (hunk 16 above)
Comment (sumyapp) [inline app/Console/cake.php:38]: **[staging][code_sniffer]**
- Concat operator must not be surrounded by spaces
- Concat operator must not be surrounded by spaces

File: app/Config/acl.php (hunk 17 above)
Comment (sumyapp) [inline app/Config/acl.php:118]: **[staging][code_sniffer]**
- Array key not aligned correctly; expected 18 spaces but found 4

File: app/Config/acl.php (hunk 18 above)
Comment (sumyapp) [inline app/Config/acl.php:119]: **[staging][code_sniffer]**
- Array key not aligned correctly; expected 18 spaces but found 4

File: app/Config/acl.php (hunk 19 above)
Comment (sumyapp) [inline app/Config/acl.php:141]: **[staging][code_sniffer]**
- Array key not aligned correctly; expected 20 spaces but found 4
- Multi-line array contains a single value; use single-line array instead

File: app/Config/acl.php (hunk 20 above)
Comment (sumyapp) [inline app/Config/acl.php:143]: **[staging][code_sniffer]**
- Closing parenthesis not aligned correctly; expected 15 space(s) but found 4

File: app/Config/Schema/db_acl.php (hunk 3 above)
Comment (sumyapp) [inline app/Config/Schema/db_acl.php:26]: **[staging][phpmd]**
- The property $aros_acos is not named in camelCase.

File: app/Config/Schema/db_acl.php (hunk 4 above)
Comment (sumyapp) [inline app/Config/Schema/db_acl.php:35]: **[staging][phpmd]**
- Avoid unused parameters such as '$event'.

File: app/Config/Schema/db_acl.php (hunk 5 above)
Comment (sumyapp) [inline app/Config/Schema/db_acl.php:46]: **[staging][phpmd]**
- Avoid unused parameters such as '$event'.

File: app/Config/Schema/i18n.php (hunk 6 above)
Comment (sumyapp) [inline app/Config/Schema/i18n.php:39]: **[staging][phpmd]**
- Avoid unused parameters such as '$event'.

File: app/Config/Schema/i18n.php (hunk 7 above)
Comment (sumyapp) [inline app/Config/Schema/i18n.php:50]: **[staging][phpmd]**
- Avoid unused parameters such as '$event'.

File: app/Config/Schema/sessions.php (hunk 8 above)
Comment (sumyapp) [inline app/Config/Schema/sessions.php:27]: **[staging][phpmd]**
- The property $cake_sessions is not named in camelCase.

File: app/Config/Schema/sessions.php (hunk 9 above)
Comment (sumyapp) [inline app/Config/Schema/sessions.php:38]: **[staging][phpmd]**
- Avoid unused parameters such as '$event'.

File: app/Config/Schema/sessions.php (hunk 10 above)
Comment (sumyapp) [inline app/Config/Schema/sessions.php:49]: **[staging][phpmd]**
- Avoid unused parameters such as '$event'.

File: app/Controller/PagesController.php [hunk 27]
```diff
@@ -28,54 +28,72 @@
  * @package       app.Controller
  * @link http://book.cakephp.org/2.0/en/controllers/pages-controller.html
  */
-class PagesController extends AppController {
+class PagesController extends AppController
```
Comment (sumyapp) [inline app/Controller/PagesController.php:31]: **[staging][phpmd]**
- The class PagesController has 18 public methods. Consider refactoring PagesController to keep number of public methods under 10.
- The property $顧客名 is not named in camelCase.

File: app/Controller/PagesController.php [hunk 28]
```diff
@@ -84,79 +102,82 @@ public function display17() {
  * @throws NotFoundException When the view file could not be found
  *	or MissingViewException in debug mode.
  */
-	public function display() {
-    $eeeeeeeeee;
-    $aaaaaaaaaa;
-    $bbbbbbbbbb;
-    $aaaaaaaaaa;
+    public function display()
```
Comment (sumyapp) [inline app/Controller/PagesController.php:105]: **[staging][phpmd]**
- The variable $title_for_layout is not named in camelCase.
- The variable $title_for_layout is not named in camelCase.

File: app/Controller/PagesController.php [hunk 29]
```diff
@@ -84,79 +102,82 @@ public function display17() {
...
+    {
+        $eeeeeeeeee;
```
Comment (sumyapp) [inline app/Controller/PagesController.php:107]: **[staging][phpmd]**
- Avoid unused local variables such as '$eeeeeeeeee'.

File: app/Controller/PagesController.php (hunk 27 above)
Comment (ghost) [inline app/Controller/PagesController.php:31]: **[phpmd]**
- The class PagesController has 18 public methods. Consider refactoring PagesController to keep number of public methods under 10.
- The property $顧客名 is not named in camelCase.

File: app/Controller/PagesController.php (hunk 28 above)
Comment (ghost) [inline app/Controller/PagesController.php:105]: **[phpmd]**
- The variable $title_for_layout is not named in camelCase.
- The variable $title_for_layout is not named in camelCase.

**Pull request closed without merging**